magprime/templates/regextra.html

```diff
@@ -1,6 +1,7 @@
 <script type="text/javascript">
     $(function () {
         $('#extra_donation').insertAfter($.field('amount_extra').parents('.form-group'));
```

Comment:
In general, we should probably indent HTML code which falls inside ``{% if %}`` blocks.  One pages where there are tons of different nested levels of loops, ifs, and HMTL tags it makes it easier to tell which content is associated with which if statement,